e2e/cypress/integration/messaging/reactions_spec.js

```diff
@@ -0,0 +1,183 @@
+
+                // # Hover over the post again and observe the `+` icon next to your reaction
+                cy.get(`#addReaction-${postId}`).should('exist').click({force: true});
+
+                // # Click the `+` icon and select a different reaction
+                cy.get('.emoji-picker__items #emoji-1f643').wait(TIMEOUTS.HALF_SEC).click();
+
+                // * Two reactions are added to the message in the expanded RHS
+                cy.get(`#rhsPost_${postId}`).within(() => {
+                    cy.findByLabelText('reactions').should('be.visible');
+                    cy.findByLabelText('remove reaction scream').should('be.visible');
+                    cy.findByLabelText('remove reaction upside down face').should('be.visible');
+                });
+
+                // # Close RHS
+                cy.closeRHS();
+            });
+        });
+    });
+
+    it('MM-T2195 Emoji reaction - not available on system message Flag - not available on system message Pin - not available on system message Can delete your own system message', () => {
+        // # Click add a channel description
+        cy.get('#channelHeaderDescription button').click();
+
+        // # Add or update a channel header
+        cy.get('#editChannelHeaderModalLabel').should('be.visible');
+        cy.get('textarea#edit_textbox').should('be.visible').type('This is a channel description{enter}');
+
+        cy.getLastPostId().then((postId) => {
+            // * Emoji reaction - not available on system message
+            cy.get(`#post_${postId}`).trigger('mouseover', {force: true});
+            cy.wait(TIMEOUTS.HALF_SEC).get(`#CENTER_reaction_${postId}`).should('not.exist');
+
+            // * Flag - not available on system message
+            cy.get(`#post_${postId}`).trigger('mouseover', {force: true});
+            cy.wait(TIMEOUTS.HALF_SEC).get(`#CENTER_flagIcon_${postId}`).should('not.exist');
+
+            // * Pin - not available on system message
+            cy.get(`#post_${postId}`).should('be.visible').within(() => {
+                cy.get('.post-menu').should('be.visible').within(() => {
+                    return cy.findByText('Pin to Channel').should('not.exist');
+                });
+            });
+
+            // * If permissions allow, can click [...] >
+            cy.clickPostDotMenu(postId);
+
+            // # Delete to delete system message
+            cy.get(`#delete_post_${postId}`).click();
+
+            // * Check that confirmation dialog is open.
+            cy.get('#deletePostModal').should('be.visible');
+
+            // # Confirm deletion.
+            cy.get('#deletePostModalButton').click();
+        });
+    });
+
+    it('MM-T2196 Emoji reaction - not available on ephemeral message Flag - not available on ephemeral message Pin - not available on ephemeral message Timestamp - not a link on ephemeral message Can close ephemeral message', () => {
```

Comment:
Same for this test case please - replace `flag` with `save` 

I've updated the test case in TM4J

Reply:
@lindy65 I already updated it, you've somehow you might seen the old commit :) Sometimes GitHub doesn't automatically update the page.